src/queue.ts

```diff
@@ -81,7 +74,16 @@ export class QueueImpl implements Queue {
       .then((entries) => {
         if (entries.length > 0) {
           const { url, body, timestamp, statusCode, attemptCount } = entries[0];
-          return fetchFn(url, body, createHeaders(attemptCount, statusCode))
+          debug(
```

Comment:
how are these debug statements stripped out for prod build?

Reply:
The debug statements aren't extracted, there's a debug flag to enable logging in prod build